playbooks/common/openshift-cluster/initialize_openshift_version.yml

```diff
@@ -0,0 +1,29 @@
+---
+# NOTE: requires openshift_facts be run
+- name: Determine openshift_version to configure on first master
```

Comment:
Ideally, I would like to see this pushed down into a role dependency, where we use delegate_to for getting the data for the first master (if we aren't the first master)